src/components/review/ReviewsList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+import WriteReview from './WriteReview';
+import { Alert } from '@mui/material';
+
+const AlbumReviews = ({ albumId }) => {
+  const [reviews, setReviews] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    fetchAlbumReviews();
+  }, [albumId]);
+
+  const fetchAlbumReviews = async () => {
+    try {
+      const response = await axiosInstance.get(`/reviews/album/${albumId}`);
+      console.log(response.data.allProductReviews);
+
+      const { allProductReviews } = response.data;
+      setReviews(allProductReviews);
+      setLoading(false);
+    } catch (error) {
+      console.error('Error fetching album reviews:', error);
+      setError('Error fetching album reviews. Please try again later.');
+      setLoading(false);
+    }
+  };
+
+  // Function to refresh reviews
+  const refreshReviews = () => {
+    // Simply call the fetchAlbumReviews function to refresh
+    fetchAlbumReviews();
+  };
+
+  return (
+    <div>
+      <h2>Album Reviews</h2>
+      {error && <Alert severity="error">{error}</Alert>}
+
+      {loading && <p>Loading reviews...</p>}
+
+      {reviews?.length ? (
+        <ul>
+          {reviews.map((review) => (
```

Comment:
It might be nice to show when the review was written and the user name of the reviewer. But these could maybe be extra things to implement in a separate PR, just an idea.

Reply:
@akosasante great idea . will implement it in another PR